repos/system_upgrade/el7toel8/actors/peseventsscanner/actor.py

```diff
@@ -15,9 +16,9 @@ class PesEventsScanner(Actor):
     """
 
     name = 'pes_events_scanner'
-    consumes = (InstalledRedHatSignedRPM, RepositoriesBlacklisted,)
-    produces = (RpmTransactionTasks, RepositoriesSetupTasks, Report,)
+    consumes = (InstalledRedHatSignedRPM, RpmTransactionTasks, RepositoriesBlacklisted)
+    produces = (PESRpmTransactionTasks, RepositoriesSetupTasks, Report,)
     tags = (IPUWorkflowTag, FactsPhaseTag)
 
     def process(self):
-        scan_events('/etc/leapp/files/pes-events.json')
```

Comment:
pes_events_scanner('/etc/leapp/files/pes-events.json') is more self-descriptive

Reply:
Changed.